IsantePlus.hbm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced